### PR TITLE
build refinements 20190617

### PR DESCRIPTION
**Pull request by algobolson**

### CODE REVIEW

File: scripts/build_release.sh [hunk 1]
```diff
@@ -172,7 +179,9 @@ EOF
 dpkg -l >>"${STATUSFILE}"
 gpg --clearsign "${STATUSFILE}"
 gzip "${STATUSFILE}.asc"
-aws s3 cp --quiet "${STATUSFILE}.asc.gz" "s3://algorand-devops-misc/buildlog/${RSTAMP}/${STATUSFILE}.asc.gz"
+if [ ! -z "${S3_PREFIX_BUILDLOG}" ]; then
+    aws s3 cp --quiet "${STATUSFILE}.asc.gz" "${S3_PREFIX_BUILDLOG}/${RSTAMP}/${STATUSFILE}.asc.gz"
```
Comment (Karmastic) [inline scripts/build_release.sh]: any reason this is silent but buildlog copy below (in build_release_local.sh) is not?

Comment (algobolson) [reply scripts/build_release.sh]: this script is expected to run as a script and be recorded to a log file. `aws s3 cp` without --quiet prints out status a couple times a second and floods that log file. `build_release_local.sh` is at this time really more of a copy&paste source of things to do to run the build that get run in the build engineer's terminal.

File: scripts/build_packages.sh [hunk 2]
```diff
@@ -131,13 +131,5 @@ for var in "${VARIATION_ARRAY[@]}"; do
 	    cp -p *.deb ${PKG_ROOT}/${GATE_PREFIX}algorand_${CHANNEL}_${PKG_NAME}_${FULLVERSION}.deb
             popd
         fi
-
-        # For now, we only deploy packages with telemetry (so we can transition smoothly).
-        # Copy xxx_channel_yyy -> xxx_channel-telem_yyy
-        if [[ ("${var}" = "") && (! -z ${TRANSITION_TELEMETRY_BUILDS}) ]]; then
```
Comment (winder) [inline scripts/build_packages.sh]: Have we transitioned? Can we remove this flag entirely?

**Comment by Karmastic** [comment]
Yes we have transitioned several builds ago. Anyone still on an old build
should reinstall or just specify -c stable

**Pull request merged**